### ISSUE TITLE
Custom sorting doesn't work for type: select

### ISSUE THREAD

**Issue by jarek-gilewski** [bolt/bolt#6561]
I have block of elements with contacts and I would like to show them in specific order on multiple pages in page footer - different contacts based on different pages. Contacts are visible on page and also in admin panel but custom sorting works only once - I mean when I change order of contacts and save all is fine. When I refresh admin panel and save (even different field) contacts are sorted in alphabetical order.

```
selectentry:
            label: Contact
            type: select
            values: contact/title
            group: Contacts
            autocomplete: true
            multiple: true
            sortable: true
```

Details
-------

 - Relevant Bolt Version: [3.2.9]
 - Install type: [Zip/tarball install]
 - PHP version: [5.6.16]
 - Used web server: [ Apache ] version [ Apache/2 ]
 - For UX/UI issues: [ all browsers ]

**Comment by bobdenotter** [comment]
For reference, a complete contenttype to reproduce this is: 

```
bars:
    name: Bars
    singular_name: Bar
    fields:
        title:
            type: text
            class: large
        slug:
            type: slug
            uses: [ title ]
        selectentry:
            type: select
            values: entries/id,title
            postfix: "Select an entry"
            autocomplete: true
            sortable: true
            multiple: true
```

This is related to #6355

**Comment by bobdenotter** [comment]
Fixed for Bolt 3.2.x in #6610. We're looking into Bolt 3.3.x.

**Comment by bobdenotter** [comment]
Did some testing, currently broken for 3.3.. @rossriley, could you look into this? 

![http://g.recordit.co/v4gpRiRWP3.gif](http://g.recordit.co/v4gpRiRWP3.gif)

**Comment by bobdenotter** [comment]
Fixed in #6610 and #6638
